src/app.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
 import './app.css';
+import React from 'react';
 import Display from './components/Display';
 import ButtonPanel from './components/ButtonPanel';
```

Comment:
- This component is supposed to be in the `/src/components/` folder........instructions of milestone 2 (please make that possible)
- `calculate.js`  should be  imported in App component

Reply:
Yes, that's true but the repo would look clumsy especially now that besides each component I have test code residing beside it.